src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -41,39 +42,21 @@ export const addAggregatorAndDestinationToMemo = ({
     affiliateBps,
   ] = quotedMemo.split(MEMO_PART_DELIMITER)
 
+  const maxMemoSize = (() => {
+    if (sellChainId === bchChainId) return BCH_MAXIMUM_BYTES_LENGTH
+    if (isUtxoChainId(sellChainId)) return UTXO_MAXIMUM_BYTES_LENGTH
+    return Infinity
+  })()
+
```

Comment:
Extract me as a function for reuse? i.e https://github.com/shapeshift/web/pull/7006/files#r1620367600

Reply:
Done in 03978b18e16